.github/workflows/check-all.yml

```diff
@@ -0,0 +1,42 @@
+# Check all crates.
+#
+# This workflow checks that `Cargo.lock` is configured sufficiently so that each
+# crate can be compiled on its
+# own.
+name: check-all
+
+permissions:
+  contents: read
+
+on:
+  pull_request:
+    paths:
+      - Cargo.lock
+      - .github/workflows/check-all.yml
+
+env:
+  CARGO_ACTION_FMT_VERSION: v0.1.3
+  CARGO_INCREMENTAL: 0
+  CARGO_NET_RETRY: 10
+  RUST_BACKTRACE: short
+  RUSTUP_MAX_RETRIES: 10
+
+jobs:
+  check-all:
+    timeout-minutes: 20
+    runs-on: ubuntu-latest
+    container:
+      image: docker://rust:1.56.1-buster
+    steps:
+      - run: |
+          curl --proto =https --tlsv1.3 -vsSfLo /usr/local/bin/cargo-action-fmt "https://github.com/olix0r/cargo-action-fmt/releases/download/release%2F${CARGO_ACTION_FMT_VERSION}/cargo-action-fmt-x86_64-unknown-linux-gnu"
+          chmod 755 /usr/local/bin/cargo-action-fmt
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
+      - run: cargo fetch
+      - run: |
+          cargo check --frozen \
+            --workspace \
+            --all-targets \
+            --exclude=linkerd-meshtls-boring \
```

Comment:
why is `meshtls-boring` excluded from this?

Reply:
it needs external deps (clang & cmake) to build; and it's not actually enabled in builds right now.

Reply:
ah, got it! carry on then!